software/configs/apertus.sh

```
@@ -0,0 +1,2 @@
+#!/bin/bash
+for f in /opt/axiom-firmware/software/scripts/*.func; do source $f; done
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)